packages/expo-image-picker/package.json

```diff
@@ -33,11 +33,14 @@
   "license": "MIT",
   "homepage": "https://docs.expo.io/versions/latest/sdk/imagepicker/",
   "jest": {
-    "preset": "expo-module-scripts/ios"
+    "preset": "expo-module-scripts"
   },
   "unimodulePeerDependencies": {
     "@unimodules/core": "*"
   },
+  "dependencies": {
+    "expo-permissions": "*"
```

Comment:
isn't this a `unimodulePeerDependencies` thing?

Reply:
not actually used for anything

Reply:
@EvanBacon - just to clarify - if we're importing other unimodule from the JS code then the dependency needs to be under `dependencies` so it gets installed by npm/yarn.

Reply:
@brentvatne I think you should have add `unimodules-permissions-interface` here as well 🤔